src/components/Backoffices/Dashboard/EmployeeAwards.tsx

```
@@ -0,0 +1,68 @@
+    },
+  });
+
+  return (
+    <div className="border border-gray-200 shadow-sm">
+      <div className="border-b border-gray-200 p-3 bg-green-300">
+        <h3 className="text-lg font-semibold">Daily Top Employees</h3>
+      </div>
+      <div className="overflow-y-auto h-40">
+        {Object.entries(crewDailyPurchases)?.map(([crewName, payment]) => {
+          return (
+            <div className="flex items-center justify-between gap-3 border-b border-gray-200 py-2 hover:bg-gray-200 px-3">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)